src/sparseml/modifiers/logarithmic_equalization/base.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from sparseml.modifiers.smoothquant import SmoothQuantModifier
+
+
+__all__ = ["LogarithmicEqualizationModifier"]
+
+
+class LogarithmicEqualizationModifier(SmoothQuantModifier):
+    """
+     Implements the Logarithmic Equalization Algorithm from
+     https://arxiv.org/abs/2308.15987.
+     This modifier performs a channel-wise smoothing of outliers in activations,
+     making them easier to quantize by reducing the dynamic range. The smoothing is
+     offset by applying the inverse operation to the next layer of weights, making
+     the weights slightly more difficult to quantize.
+
+     Because this modifier manipulates the weights of the model, it can only be
+     used in one-shot and not during training. Activation ranges are determined by
```

Comment:
It's not clear to me why this modifier cannot be used during training. Since its application is mathematically neutral, it seems like it can also be applied between training steps (though it may require adding modules to the network, and also  may not be particularly helpful in any way).

Can probably resolve this by changing "can only be used" -> "should only be used".

Reply:
Fixed